src/HotChocolate/Core/test/Types.Tests/Types/Relay/IdAttributeTests.cs

```diff
@@ -47,46 +68,28 @@ await SchemaBuilder.New()
         }
 
         [Fact]
-        public async Task PolyId_On_Arguments()
+        public async Task InterceptedId_On_Arguments()
         {
             // arrange
-            var idSerializer = new IdSerializer();
-            var intId = 1;
-            var stringId = "abc";
-            Guid guidId = Guid.Empty;
+            var guidId = new Guid("26a2dc8f-4dab-408c-88c6-523a0a89a2b5");
 
             // act
             IExecutionResult result =
                 await SchemaBuilder.New()
                     .AddQueryType<Query>()
                     .AddType<FooPayload>()
-                    .TryAddTypeInterceptor<PolymorphicGlobalIdsTypeInterceptor>()
                     .Create()
                     .MakeExecutable()
                     .ExecuteAsync(
                         QueryRequestBuilder.New()
-                            .SetQuery(
-                                @"query foo ($intId: ID! $stringId: ID! $guidId: ID!) {
-                                    intId(id: $intId)
-                                    intIdList(id: [$intId])
-                                    stringId(id: $stringId)
-                                    stringIdList(id: [$stringId])
-                                    guidId(id: $guidId)
-                                    guidIdList(id: [$guidId $guidId])
-                                }")
-                            .SetVariableValue("intId", intId)
-                            .SetVariableValue("stringId", stringId)
-                            .SetVariableValue("guidId", guidId.ToString())
+                            .SetQuery(@"query foo {
+                                interceptedId(id: 1)
+                                interceptedIds(id: [1, 2])
+                            }")
```

Comment:
I removed the polymorphic id formatter and instead just did a quick inline descriptor that injects a formatter that does just enough to prove that the GlobalIdFormatter can be intercepted by it.